db/migrate/20150120095012_create_categories.rb

```diff
@@ -2,6 +2,7 @@ class CreateCategories < ActiveRecord::Migration
   def change
     create_table :categories do |t|
       t.string :name
+      t.string :link
```

Comment:
the url for the category health & fitness is www.pinterest.com/category/health_fitness rather than the name of the category 'Health & Fitness'. The link bit is different